tests/acceptance/tests/paypal-account.setup.ts

```diff
@@ -0,0 +1,44 @@
+import { test } from '@fixtures/AcceptanceTest';
+import { expect } from '@shopware-ag/acceptance-test-suite';
+import type * as PayPal from 'SwagPayPal/types';
+
+test('PayPal Account setup', {}, async ({ AdminApiContext }) => {
+    const resetResponse = await AdminApiContext.post('./_action/paypal/save-settings', {
+        data: {
+            null: {
+                'SwagPayPal.settings.clientId': null,
+                'SwagPayPal.settings.clientSecret': null,
+                'SwagPayPal.settings.merchantPayerId': null,
+                'SwagPayPal.settings.clientIdSandbox': null,
+                'SwagPayPal.settings.clientSecretSandbox': null,
+                'SwagPayPal.settings.merchantPayerIdSandbox': null,
+                'SwagPayPal.settings.sandbox': false,
```

Comment:
Just a question since I'm not the cleverestest with FE or Acceptance testing. CAN this be written as follows:

```ts
null: {
    SwagPayPal: {
        settings: {
            clientId: null,
            clientSecret: null,
            merchantPayerId: null,
            clientIdSandbox: null,
            clientSecretSandbox: null,
            merchantPayerIdSandbox: null,
            sandbox: false
        }
    }
}
```

Reply:
No, you can't, it's a flat object - although system config keys look like they work like this, they don't